lib/arrow_web/views/disruption_view/calendar.ex

```diff
@@ -42,7 +43,7 @@ defmodule ArrowWeb.DisruptionView.Calendar do
     |> Enum.map(fn {event_start, event_end} ->
       %{
         title: source_label,
-        classNames: "route-#{route_id}",
+        classNames: "#{row_status(row_approved)}-route-#{route_id}",
```

Comment:
suggestion: what do you think about having separate classes for the approval status and the route? Something like:
```suggestion
        classNames: "{row_status_class(row_approved)} route-#{route_id}",
```

While it doesn't make much difference here (you still need all the same styles) it's more flexible in the future if we need to target them separately.

Reply:
It appears that implementing your suggested change doesn't produce the expected color features in the calendar. Everything is set to the default blue. Am I missing something?

Reply:
What he means is instead of having a combined class that you then specifically style, like:

```
html:
<div class="approved-route-Blue"> ... </div>

css:
.approved-route-Blue { .... }
```

to instead add two separate classes, which can then be selected in the CSS by naming both:

```
html:
<div class="approved route-Blue"> ... </div>

css:
.approved.route-Blue { .... }
```

In the HTML, note that spaces delimit different classes, and an element can have multiple classes on it. In the CSS, note that you can style "elements with both this class1 and that class2" by using the `.class1.class2` selector (no space between the classes). (And in the case here, I think it would be `.fc-event.pending-status.route-Blue`.)

Right now, there's not much difference between the approaches, but by making `.pending-status` its own class, we could potentially do something like:

```css
.approved-status { font-weight: bold }
```

and then that would make all the approved events' bold, without having to individually style it for every route.